src/main/java/domain/card/CardFactory.java

```diff
@@ -14,7 +13,7 @@
         for (Symbol symbol : symbols) {
             createByType(cards, symbol);
         }
-        return Collections.unmodifiableList(cards);
+        return cards;
```

Comment:
The reason It gave _the card as an unmodifiable list_ is because after mixing it randomly in a real game, there will be no changes in the card during that game. Therefore, during the game, **you must have an unmodifiable card and mix randomly before return**